deepbots/__init__.py

```diff
@@ -1 +1 @@
-__version__ = "0.0.0.11-pre"
+__version__ = "0.1.3.dev2"
```

Comment:
```suggestion
__version__ = "0.1.3-dev2"
```
We have "0.1.3-dev2" [here](https://github.com/aidudezzz/deepbots/blob/32a23b2ccfd9dcdab6dc1dba1e3cbe8772e84e4e/.bumpversion.cfg#L2)

Reply:
Sorry about that, fixed it.